addons/binding/org.openhab.binding.mihome/README.md

```diff
@@ -353,6 +354,14 @@ then
     <ACTION>
 end
 
+rule "Unpair a device from the gateway"
```

Comment:
Should this be cleaned up?

Reply:
Yes, thanks! I even found a few more spots in the readme to tune...